tests/test_features/test_jointplot.py

```diff
@@ -0,0 +1,65 @@
+# tests.test_features.test_jointplot
+# Test the JointPlotVisualizer
+#
+# Author:   Prema Damodaran Roman
+# Created:  
+#
+# Copyright (C) 2017 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: test_jointplot.py
+
+"""
+Test the JointPlotVisualizer
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import unittest
+import numpy as np
+import numpy.testing as npt
+
+from tests.dataset import DatasetMixin
+from yellowbrick.features.jointplot import *
+
+##########################################################################
+## JointPlotVisualizer Tests
+##########################################################################
+
+class JointPlotTests(unittest.TestCase, DatasetMixin):
+
+	X = np.array([1, 2, 3, 5, 8, 10])
```

Comment:
Potentially you could update this to a 2 dimensional array and then test all the X[i], y combinations since this is closer to what a user would have to do with this plot. You could also stress the joint plot a bit by using np.rand and having slightly larger arrays -- this might catch edge cases and errors that you didn't expect.